portfolio/data-servlet/main/webapp/script.js

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Adds a random greeting to the page.
+ */
+function addRandomQuote() {
+  const quotes =
+      ['I like that boulder. That is a nice boulder.', 'I bet you have never seen a donkey fly!', 'She called me a noble steed.', 'Better out than in, I always say.'];
+
+  // Pick a random greeting.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+  // Add it to the page.
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+}
+
+
+
+var slideIndex = 1;
```

Comment:
`var` is totally fine here. Generally consider using `let` instead of `var` to avoid issues with hoisting and scoping.